### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by autoih**

### CODE REVIEW

File: tensorflow/python/ops/array_ops.py [hunk 1]
```diff
@@ -3962,7 +3962,7 @@ def reverse_sequence(input,
   >>> seq_lengths = [7, 2, 3, 5]
   >>> input = [[1, 2, 3, 4, 5, 0, 0, 0], [1, 2, 0, 0, 0, 0, 0, 0],
   ...          [1, 2, 3, 4, 0, 0, 0, 0], [1, 2, 3, 4, 5, 6, 7, 8]]
-  >>> output = reverse_sequence(input, seq_lens, seq_dim=1, batch_dim=0)
+  >>> output = reverse_sequence(input, seq_lengths, seq_dim=1, batch_dim=0)
```
Comment (jaingaurav) [inline tensorflow/python/ops/array_ops.py]: Could you please update the code to call `tf.reverse_sequence` instead of just `reverse_sequence`

Comment (jaingaurav) [reply tensorflow/python/ops/array_ops.py]: Also it seems like this should be `seq_axis` instead of `seq_dim`

Comment (jaingaurav) [reply tensorflow/python/ops/array_ops.py]: @yashk2810: Is there a reason these errors weren't caught in our doctects?

Comment (yashk2810) [reply tensorflow/python/ops/array_ops.py]: So I investigated and seems like that doctest is not being run at all. 

The reason has to do with the lines below it

```
reverse_sequence.__doc__ = deprecation.rewrite_argument_docstring(
    deprecation.rewrite_argument_docstring(
        gen_array_ops.reverse_sequence.__doc__, "batch_dim", "batch_axis"),
    "seq_dim", "seq_axis")
```

`deprecation` replaces the docstring with the docstring of `gen_array_ops.reverse_sequence` which doesn't seem to have the `>>>` (carets).

Hence, it doesn;t get tested. We will need to fix that for it to start getting tested.

**Review by jaingaurav — APPROVED** [review]

File: tensorflow/python/ops/array_ops.py (hunk 1 above)
Comment (autoih) [reply tensorflow/python/ops/array_ops.py]: Thanks @jaingaurav and @yashk2810. I've updated the args.

**Pull request merged**